chemprop/models/mpn.py

```diff
@@ -170,7 +213,7 @@ def __init__(self,
         self.reaction = args.reaction
         self.reaction_solvent = args.reaction_solvent
         self.atom_fdim = atom_fdim or get_atom_fdim(overwrite_default_atom=args.overwrite_default_atom_features,
-                                                     is_reaction=(self.reaction or self.reaction_solvent))
+                                                    is_reaction=(self.reaction or self.reaction_solvent))
         self.bond_fdim = bond_fdim or get_bond_fdim(overwrite_default_atom=args.overwrite_default_atom_features,
                                                     overwrite_default_bond=args.overwrite_default_bond_features,
                                                     atom_messages=args.atom_messages,
```

Comment:
using logical `or` with primitive data is tenuous:
```python
>>> None or 2
2
>>> 0 or 2
2
```
I'd argue the intention with the second case was to get `0` out. It's best to be explicit here.

Reply:
Those two parameters are with boolean values. I replace the original line with `is_reaction=self.reaction if self.reaction is not False else self.reaction_solvent`.